src/js/frontend/src/components/meetingData.js

```diff
@@ -0,0 +1,133 @@
+let d3 = require('d3');
+let api = require('./api');
+
+let meetingData = {
+  sampleData: {
```

Comment:
Yes, so that the demo URL mentioned in the PR description works.  It's useful for developing / testing the frontend without a backend.